Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-AlertsAndActionSheets.swift

```diff
@@ -112,6 +118,10 @@ struct AlertAndSheetView: View {
             )
         }
       }
+      .alert(
```

Comment:
We've found that nested alert and sheet view modifiers don't work, at least in most versions of SwiftUI: https://twitter.com/mbrandonw/status/1293909768927510529

Does the above `Alert` button still somehow show `alert` state? What version of Xcode and iOS simulator are you running?

Reply:
I don't know what button you are referring to. But when resultAlert is not nil, this alert shows.
Currently I'm using Xcode 11.3 and I tested on real device(iPhone Xs Max)